catalog.json

```diff
@@ -1,18 +1,18 @@
 {
   "id": "8478b533-2b59-4007-8494-2feec5970f94",
   "name": "cdn-route",
-  "description": "Custom URL, CDN, and free TLS certificates",
+  "description": "Custom domain, CDN caching, and TLS certificates",
   "bindable": true,
   "metadata": {
-    "displayName": "Content Distribution Network",
-    "documentationUrl": "https://cloud.gov/docs/services/cdn-route"
+    "displayName": "CDN Route",
```

Comment:
I asked Navigator folks how they'd like to proceed on this, since this seems like this might need more discussion/decisionmaking rather than just a little incremental PR. :)